exec/java-exec/src/main/resources/rest/query/query.ftl

```diff
@@ -74,7 +74,7 @@
       <div style="display: inline-block"><label for="query">Query</label></div>
       <div style="display: inline-block; float:right; padding-right:5%"><b>Hint: </b>Use <div id="keyboardHint" style="display:inline-block; font-style:italic"></div> to submit</div>
       <div id="query-editor-format"></div>
-      <input class="form-control" type="hidden" id="query" name="query"/>
+      <input class="form-control" type="hidden" id="query" name="query" autofocus/>
```

Comment:
For some reason this doesn't seem to work. I can see cursor in query textbox but it's not focussed. I have to click the query textbox to type any query in it.

Reply:
Made the change, please review again.

Thanks.